src/dotnet/Chat.UI.Blazor/Components/ChatView/ChatView.razor.cs

```diff
@@ -127,14 +123,17 @@ private async Task MonitorVisibleKeyChanges(CancellationToken cancellationToken)
             }
     }
 
-    private async Task<VirtualListData<ChatMessageModel>> GetMessages(
+    async Task<VirtualListData<ChatMessageModel>> IVirtualListDataSource<ChatMessageModel>.GetData(
         VirtualListDataQuery query,
+        VirtualListData<ChatMessageModel> oldData,
         CancellationToken cancellationToken)
     {
         await WhenInitialized;
 
         var chat = Chat;
         var chatId = chat.Id.Value;
+        var currentAuthor = await ChatAuthors.Get(Session, chatId, cancellationToken);
```

Comment:
I was told that we do not almost use `current` as a useless prefix

Reply:
OK